File: backend/ViewModel/Ingredient.cs
namespace backend.ViewModel;

public class Ingredient
{
    public int Id { get; set; }
    public IngredientType IngredientType { get; set; }
    public bool IsHealthy { get; set; }
    public required string Name { get; set; }
    public float Price { get; set; }
    public required List<Pancake> Pancakes { get; set; }
}

File: frontend/src/app/models/Ingredient.ts
import {Pancake} from "./Pancake";
import {IngredientType} from "./IngredientType";

export interface Ingredient {

  id: number;
  ingredientType: IngredientType;
  isHealthy: boolean;
  name: string;
  price: number;
  pancakes: Pancake[];
}

